### ISSUE TITLE
python3 test failure

### ISSUE THREAD

**Issue by opoplawski** [Unidata/netcdf4-python#523]
Now seeing this with Fedora Rawhide, only with the python3 tests.
~~~~~
+ /usr/bin/python3 run_all.py
..........................E.E................................................not running tst_unicode.py ...

netcdf4-python version: 1.2.2
HDF5 lib version:       1.8.16
netcdf lib version:     4.4.0
numpy version           1.11.0b2

======================================================================
ERROR: runTest (tst_multifile.VariablesTestCase)
testing multi-file dataset access
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/netcdf4-python-1.2.2rel/test/tst_multifile.py", line 36, in setUp
    dat[:] = data[nx1:nx2]
  File "/usr/lib64/python3.5/site-packages/numpy/ma/core.py", line 3116, in __getitem__
    dout = self.data[indx]
IndexError: invalid slice

======================================================================
ERROR: runTest (tst_multifile2.VariablesTestCase)
testing multi-file dataset access
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/netcdf4-python-1.2.2rel/test/tst_multifile2.py", line 37, in setUp
    dat[:] = data[nx1:nx2]
  File "/usr/lib64/python3.5/site-packages/numpy/ma/core.py", line 3116, in __getitem__
    dout = self.data[indx]
IndexError: invalid slice

----------------------------------------------------------------------
~~~~~

**Comment by jswhit** [comment]
OK, those are both fixed now in pull request #522

**Comment by opoplawski** [comment]
Thanks.  With that it now builds fine.
